src/Roslyn.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
 # Visual Studio 14
-VisualStudioVersion = 14.0.22415.0
+VisualStudioVersion = 14.0.22512.0
```

Comment:
I don't believe any of your changes require the solution file to be modified.  Can you exclude these changes from the pull request?